src/cli.c

```diff
@@ -425,74 +451,56 @@ static int import_keys(user_options_t *options)
         }
     }
 
-    if (!user) {
-        printf("Bridge username (email): ");
-        get_input(user_input);
-        num_chars = strlen(user_input);
-        user = calloc(num_chars + 1, sizeof(char));
-        if (!user) {
-            status = 1;
-            goto clear_variables;
-        }
-        memcpy(user, user_input, num_chars * sizeof(char));
-    }
+    if (!enc_access_str) {
+        enc_access_str = calloc(BUFSIZ, sizeof(char));
 
-    if (!pass) {
-        printf("Bridge password: ");
-        pass = calloc(BUFSIZ, sizeof(char));
-        if (!pass) {
-            status = 1;
-            goto clear_variables;
-        }
-        get_password(pass, '*');
+        printf("Enter your serialized encryption key (leave blank to generate new): ");
```

Comment:
We are expecting a "serialized encryption access" here, not an encryption key, right?

Reply:
Correct, my understanding was that going forward the user-facing language for encryption access context whatever was going to be "encryption key"

Reply:
I am not sure about this... We create encryption access from the encryption key, which we derive from the passphrase. So the encryption access and the encryption key should be different things, although related.